docker/search-api/.dockerignore

```diff
@@ -9,6 +9,3 @@ nginx/conf.d
 
 # Ignore the flask instance folder
 src/instance
-
-# Ignore search-schema from the submodule
```

Comment:
There's no longer this `src/search-schema` sub directory in the submodule: https://github.com/dbmi-pitt/search-adaptor are you using an older version?

Reply:
Right: If it's no longer there, we no longer need to dockerignore it. This line doesn't do any harm, but we shouldn't keep vestigial code around either.

Reply:
Cool, I'll just delete that line from dockerignore to keep things consistent and updated to avoid future confusion then merge.